src/components/SingleItem.js

```diff
@@ -1,55 +1,126 @@
+import Link from 'next/link';
 import styled from 'styled-components';
 
 import useStore from './../hooks/useStore';
 
+const Wrapper = styled.section`
+	padding: 24px;
+`;
 const Picture = styled.img`
 	max-width: 185px;
 	border-radius: 16px 16px 16px 16px;
 	box-shadow: 1px 1px 10px 5px rgba(0, 0, 0, 0.2);
 `;
 
-const MovieHeadline = styled.p`
-	padding: 12px 8px 0 16px;
-	color: black;
+const Hl2 = styled.h2`
+	padding: 16px 0 0 0;
+	color: white;
+	overflow-wrap: break-word;
+	font-size: 36px;
+	font-style: normal;
+	font-weight: 600;
+	& span {
+		font-weight: 400;
+	}
+`;
+
+const Hl3 = styled.h3`
+	padding: 0 8px 8px 0;
+	color: white;
 	overflow-wrap: break-word;
 	font-size: 20px;
+	font-style: italic;
+	font-weight: 400;
 `;
 
-const Ho2 = styled.h2`
-	margin: 32px 16px 0 8px;
-	padding: 12px 8px 16px 8px;
-	color: black;
+const Hl4 = styled.h4`
+	padding: 24px 8px 0 0;
+	color: white;
 	overflow-wrap: break-word;
-	font-size: 24px;
+	font-size: 18px;
+	font-style: normal;
+	font-weight: 600;
+	line-height: 1.3;
+	& span {
+		font-style: normal;
+		font-weight: 200;
+	}
 `;
 
 const Article = styled.article`
 	display: flex;
-	padding: 16px;
+	padding: 24px 0;
+`;
 
-	& p {
-		padding: 0 32px;
-	}
+const Paragraph = styled.p`
+	padding: 0 32px;
+	color: white;
+	line-height: 1.5;
 `;
 
 export default function SingleItem(props) {
+	const urlSource = props.type === 'tv' ? '/tv/' : '/movie/';
 	const objData = props.data;
 
+	// playtime helper function
+	function toHoursAndMinutes(totalMinutes) {
+		const minutes = totalMinutes % 60;
+		const hours = Math.floor(totalMinutes / 60);
+
+		return `${hours}h ${padTo2Digits(minutes)}m`;
+	}
+	function padTo2Digits(num) {
+		return num.toString().padStart(2, '0');
+	}
 	const objConfiguration = useStore(state => state.objConfiguration);
 	const imagesBaseUrl = objConfiguration.secure_base_url + objConfiguration.poster_sizes[2];
+	const backdropImageUrl = objConfiguration.secure_base_url + objConfiguration.backdrop_sizes[1];
 
 	return (
-		<section>
-			<Ho2>Editor´s Pick</Ho2>
-			<MovieHeadline>{objData.title || objData.name}</MovieHeadline>
+		<Wrapper
```

Comment:
Hmmm... I personally avoid inline styling at all costs, because of seperation of concerns.

Reply:
Ah, you mean the backgroundImage. Yes, this isn't perfect but i haven't seen another possibility to do this with an dynamically loaded image + color overlay with opacity.